packages/langchain/src/openai/embedding.ts

```diff
@@ -25,22 +26,18 @@ export class AzureOpenAiEmbeddingClient extends Embeddings {
   }
 
   override async embedDocuments(documents: string[]): Promise<number[][]> {
```

Comment:
[pp] What is a bit annoying is that the parameter is named `documents`, which might lead users to believe that they can / should pass full documents, instead of chunks.

But that is on the langchain interface I guess, not on us 🤷🏻‍♂️

Reply:
Maybe we should update the readme and explicitly call it `document chunks` there?